payloads/elasticat.py

```diff
@@ -122,13 +133,30 @@ def _encode_string(s):
         return str(b64encode(s.encode()), 'utf-8')
 
 
+def valid_date_format(s):
+    try:
+        if s == 'now':
+            return s
+        date = date_parser.parse(s)
+        if date:
+            return date.strftime('%Y-%m-%dT%H:%M:%SZ')
+    except ValueError:
+        error_msg = 'supplied value "%s" is an invalid date.' % s
+        raise argparse.ArgumentTypeError(error_msg)
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('--server', default='http://127.0.0.1:8888', help='Base URL  Caldera server.')
     parser.add_argument('--es-host', default='http://127.0.0.1:9200', dest='es_host',
                         help='Base URL of ElasticSearch.')
     parser.add_argument('--index', default='*', help='ElasticSearch index pattern to search over.')
     parser.add_argument('--group', default='blue')
+    parser.add_argument('--start-time', dest='start_time', default=None, type=valid_date_format,
+                        help='Date to start searching for events. If provided without an --end-time, the query will '
+                             'search for events created until now.')
+    parser.add_argument('--end-time', dest='end_time', default='now', type=valid_date_format,
+                        help='Date to stop searching for events. Must be accompanied by a --start-time.')
     parser.add_argument('--minutes-since', dest='minutes_since', default=60, type=int,
                         help='How many minutes back to search for events.')
```

Comment:
Can you add a note to the help message that states that this will be overruled by the start/end time arguments?